server/windows/src/windows.rs

```diff
@@ -56,8 +57,14 @@ mod omsupply_service {
         let executable_path = current_exe().unwrap();
         let executable_directory = executable_path.parent().unwrap();
         set_current_dir(&executable_directory).unwrap();
-        let settings: Settings =
-            configuration::get_configuration().expect("Failed to parse configuration settings");
+        let settings: Settings = match configuration::get_configuration() {
+            Ok(settings) => settings,
+            Err(e) => {
+                eventlog::init("Application", log::Level::Error).unwrap();
+                error!("Failed to parse configuration settings: {:?}", e);
```

Comment:
if we also use log_panics::init(); at the top of omsupply_service_main, we wouldn't need to explicitly use error! here and `expect` should still work, and also it would log other unwraps and expects

Reply:
the docs for log_panics said to init after the logging was initiated, so I moved it down here. For the panic handler to run as you say, we'd need to be able to init both loggers, or un-init the event log handler. Couldn't see how to do either of those.